tungsten_ci_utils/nexus_cleanup/clean_up.groovy

```diff
@@ -0,0 +1,81 @@
+import org.sonatype.nexus.repository.storage.StorageFacet;
```

Comment:
Renamed the file to docker_cleanup.groovy since we're introducing rpm_cleanup.groovy also.